Pinta.Core/Managers/ToolManager.cs

```diff
@@ -351,7 +351,7 @@ public override int Compare (BaseTool? x, BaseTool? y)
 		private Image? tool_image;
 		private Separator? tool_sep;
 
-		private Label ToolLabel => tool_label ??= Label.New (string.Format (" {0}:  ", Translations.GetString ("Tool")));
+		private Label ToolLabel => tool_label ??= Label.New ($" {Translations.GetString ("Tool")}:  ");
```

Comment:
Yeah, it unfortunately looks like `xgettext` won't pick up calls to `Translations.GetString()` which are inside another (interpolated) string, so I think it's probably best to revert those cases except for when it's sensible to extract the GetString() call to a prior line?

Reply:
I could revert these cases or, if you prefer, I can change the bodies of the getters so that they becomes blocks, and I can assign the translation to a variable, which would solve the problem while allowing us to keep using string interpolation.

Reply:
Yeah I think reverting is probably easiest unless the getter turns into several lines

Reply:
Awesome, changes reverted